### ISSUE TITLE
COVID-19 documentation tweaks

### ISSUE THREAD

**Issue by lintool** [castorini/anserini#1045]
Comments, minor fixes: roll up in your next PR:
https://github.com/castorini/anserini/blob/master/docs/experiments-covid.md

**1.**

`${DATA_DIR}` -> add the date of the collection in the path; I assume we're going to get frequent updates, so let's not get them confused.

**2.**

```
ls ${DATA_DIR}/*.tar.gz | xargs --replace tar -zxvf {} -C ${DATA_DIR}
```

I don't think `--replace` works on Mac.

**Comment by nikhilro** [comment]
1. will do 
2. @lintool can you confirm if `-I` instead works on mac

**Comment by lintool** [comment]
nope:

```
$ ls ${DATA_DIR}/*.tar.gz | xargs -I tar -zxvf {} -C ${DATA_DIR}
xargs: illegal option -- z
usage: xargs [-0opt] [-E eofstr] [-I replstr [-R replacements]] [-J replstr]
             [-L number] [-n number [-x]] [-P maxprocs] [-s size]
             [utility [argument ...]]
```

It's only four files.... maybe it's easier just to uncompress each explicitly, rather than futz with cross-platform compatibility? Like, what about Windows?

**Comment by lintool** [comment]
Also, next iteration, add link from main `README.md`.

**Comment by nikhilro** [comment]
Resolved by #1049
